lib/request_overrider.js

```diff
@@ -379,17 +378,7 @@ function RequestOverrider(req, options, interceptors, remove, cb) {
     //  In that case we send the response as a Buffer object as that's what the client will expect.
     if (isBinaryRequestBodyBuffer && typeof responseBody === 'string') {
       //  Try to create the buffer from the interceptor's body response as hex.
-      try {
-        responseBody = Buffer.from(responseBody, 'hex')
-      } catch (err) {
-        // TODO-coverage: Add a test of this error case.
-        debug(
-          'exception during Buffer construction from hex data:',
-          responseBody,
```

Comment:
Why was this removed?

Reply:
I broke out each chante into separate commits with descriptions in the messages for the non-trivial ones. 
https://github.com/nock/nock/pull/1577/commits/de1d60a96de6230f8c0a3cc3bca8ddbaa2c51199

----
In the normal error case, that the value of response body is a string, but
not hex encoded, the attempt to create a Buffer will return a `Uint8Array`
with nothing in it.

Looking at the [Node source for Buffer.from](https://github.com/nodejs/node/blob/2817fde844de39b0e8c17c352433ee6124b5ef09/lib/buffer.js#L196), the reason errors are
thrown are:
- The first arg isn't a valid type, but we've already ensured it's a string.
- The encoding arg is not valid, but we hardcoded 'hex'.

Reply:
Thanks!